test/unit/gateways/simetrik_test.rb

```diff
@@ -0,0 +1,1262 @@
+require 'test_helper'
+require 'securerandom'
```

Comment:
minor detail: we don't need to require `securerandom` here.

Reply:
Thanks for the heads-up, it's been removed now.